server/views/pages/overview/partials/overviewTab/overviewTabContentsV2.njk

```diff
@@ -20,32 +18,33 @@
             }) }}
           </section>
       {% endif %}
-        <div class="govuk-summary-card">
+        <div class="govuk-summary-card" data-qa="goals-summary-card">
           <div class="govuk-summary-card__title-wrapper">
             <h2 class="govuk-summary-card__title govuk-!-font-size-24">Goals</h2>
           </div>
           <div class="govuk-summary-card__content">
-            <table class="govuk-table govuk-table">
-              <tbody class="govuk-table__body">
-                <tr class="govuk-table__row">
-                  <td scope="col">
-                    <span>3</span> <span class="govuk-tag govuk-tag--green">In progress</span><br><br>
-                    <p class="govuk-body govuk-!-margin-bottom-0">View in progress goals</p>
-                  </td>
-                  <td scope="col">
-                    <span>3</span> <span class="govuk-tag govuk-tag--blue">Completed</span><br><br>
-                    <p class="govuk-body govuk-!-margin-bottom-0">View completed goals</p>
-                  </td>
-                  <td scope="col">
-                    <span>3</span> <span class="govuk-tag govuk-tag--grey">Archived</span><br><br>
-                    <p class="govuk-body govuk-!-margin-bottom-0">View archived goals</p>
-                  </td>
-                </tr>
-              </tbody>
-            </table>
-            <p class="govuk-hint govuk-!-font-size-16 govuk-!-margin-top-0" data-qa="goal-last-updated-hint">
-              {{ 'params.lastUpdatedLabel' | default('Last updated on') }} {{ 'params.goal.updatedAt' | formatDate('DD MMMM YYYY') }}<span class="govuk-!-display-none-print"> by {{ 'params.goal.updatedByDisplayName' }}{{ ', ' + 'params.goal.updatedAtPrisonName' if 'params.goal.updatedAtPrisonName' }}</span>
-            </p>
+            <div class="govuk-grid">
```

Comment:
Not that I mind, but why did you change it from a govuk styled table to divs using grid css layout?

I guess semantically its more correct? It was never really tabular data, we were perhaps using the table for layout!!! ? 😬 😱 

So maybe this is more semantically correct?

Like I say, I dont mind, just intrigued as to the reason 😁

Reply:
There are vertical dividing lines which I added in after this initial code. It was tricky getting them to line up properly so I ended up having to use a lot of padding, margins etc. It was displaying weirdly in the Cypress ui and I was worried in case maybe smaller screens would have difficulty with it - probably not an issue but the grid layout seemed to solve the Cypress display weirdness.